includes/polling/functions.inc.php

```diff
@@ -634,7 +634,8 @@ function json_app_get($device, $extend, $min_version = 1)
     }
 
     //  turn the JSON into a array
-    $parsed_json = json_decode(stripslashes($output), true);
+    $output = str_replace(['\\', '\\"', "\'", '\{'], ['\\', '"', "'", '{'], $output);
```

Comment:
Shouldn't the first one be `'\\\\'` instead of `'\\'`

Reply:
Yup! Looks like that is correct. Fixed and we will see how the run goes.

```
[kitsune@vixen42]0|/home/kitsune>cat /tmp/test.php
<?php
$a = '\\\\\\"\{'."\'";
echo $a."\n";
$a = str_replace(['\\\\', '\\"', "\'", '\{'], ['\\', '"', "'", '{'], $a);
echo $a."\n";
[kitsune@vixen42]0|/home/kitsune>php /tmp/test.php
\\\"\{\'
\"{'
[kitsune@vixen42]0|/home/kitsune>
```

Reply:
Woot! All tests passed.